content/notebooks/hydrology/03_Watershed_properties.ipynb

```diff
@@ -0,0 +1,758 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    unique, counts = np.unique(grid, return_counts=True)
</pre><p>need to access <code>band_data</code> variable again here</p>

<pre class="ql-syntax" spellcheck="false">unique, counts = np.unique(grid.band_data, return_counts=True)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-landing/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>